packages/react/package.json

```diff
@@ -67,7 +67,7 @@
     "classnames": "2.3.1",
     "deepmerge": "4.2.2",
     "lodash": "4.17.21",
-    "mapbox-gl": "npm:empty-npm-package@1.0.0",
+    "mapbox-gl": "file:mapbox-resolution",
```

Comment:
This is a valid concern. I think the current folder would our dist folder. But we need to make sure this file exists in dist because it is exported through the main entry point so Rollup may not be aware of it.

Reply:
Good callout, I've updated our `package.json` to include the `lib` dir (which contains our mapbox resolution folder) in the published package. You can verify this by running `npm pack --dry-run` in the react folder and seeing that `lib/mapbox-gl/index.js` is included in the published package.

We won't need to reconfigure rollup at all since the empty package is written as cjs.